lib/improver/cli/neighbour_finding.py

```diff
@@ -192,9 +192,11 @@ def main(argv=None):
         for method in methods:
             all_methods.append(NeighbourSelection(**method).process(*fargs))
 
+        squeezed_cubes = iris.cube.CubeList([])
         for index, cube in enumerate(all_methods):
             cube.coord('neighbour_selection_method').points = index
-        result = merge_cubes(all_methods)
+            squeezed_cubes.append(iris.util.squeeze(cube))
+        result = merge_cubes(squeezed_cubes)
```

Comment:
Why did you decide to use our merge_cubes function rather than the simpler iris .merge_cubes() function?

Reply:
Given we have these wrappers, I think in principle it's safer to use those throughout, rather than a mix of the iris and improver module functionality.  For me it's about minimising complexity and maximising consistency across the code base.  Even though you're right, the extra functionality isn't needed in this particular case.

Reply:
That's fine, I just wanted to understand your thinking.